metpy/plots/skewt.py

```diff
@@ -791,20 +793,17 @@ def wind_vectors(self, u, v, **kwargs):
         return self.ax.quiver(center_position, center_position,
                               u, v, **quiver_args)
 
-    def plot_colormapped(self, u, v, c, bounds=None, colors=None, **kwargs):
+    def plot_colormapped(self, u, v, c, intervals=None, bounds=None, colors=None, **kwargs):
```

Comment:
So we need to remove `bounds` here to keep the same number of positional args...